compiler/main/driver.cpp

```diff
@@ -422,30 +350,39 @@ static void recordCodeGenStrings(int argc, char* argv[]) {
   get_version(compileVersion);
 }
 
-static void setStaticLink(const ArgumentState* state, const char* arg_unused) {
```

Comment:
If I am reading correctly, setStaticLink has been deleted. Although fLinkStyle is declared to be an "int" it seems
to me to have the flavour of an enum.  I wonder if it would be better to ensure the enum behavior rather than
drop the function?